schema.sql

```diff
@@ -1,9 +1,11 @@
 /* Database schema to keep the structure of entire database. */
-CREATE DATABASE vet_clinic;CREATE TABLE animals (
+CREATE DATABASE vet_clinic;
+  CREATE TABLE animals (
     id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
     name VARCHAR NOT NULL,
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
-);
+    weight_kg DECIMAL,
+    species VARCHAR NOT NULL
```

Comment:
Please user `ALTER TABLE` command to modify `animals` table.

Reply:
I used it but I forgot to add it the screenshot